ext/websocket/02_websocketstream.js

```diff
@@ -444,7 +445,7 @@ class WebSocketStream {
         },
         (err) => {
           this[_rid] && core.tryClose(this[_rid]);
-          this[_closed].reject(err);
+          this[_closed].reject(new WebSocketError(err));
```

Comment:
ditto; this isnt going to work since `err` isnt a string, but various kinds of errors.